### DISCUSSION TITLE
AppImage does not launch and no errors

### DISCUSSION THREAD

**Discussion by xeroine** [ppy/osu#20890] [Q&A] [answered]
### Type

Other

### Bug description

I'm running Linux Gentoo amd64, window manager is awesomewm (reproducible in other window managers) and using Xorg.

AppImage fails to launch and I get completely no output, nothing on stdout or stderr and no window shows up.
~/.local/share/osu/ directory doesn't exist/isn't created.

Since there's no logs I just put "none" in the mandatory Logs section.

What should I do to at least get more info about the issue?

### Version

2022.1022.0

### Logs

none

**Comment by bdach** [comment]
Given you're on gentoo I don't know how much assistance we can provide here. If you tweaked any useflags it may lead to catastrophic failure almost anywhere.

We could start by checking `dmesg` I guess. Is there any output there that looks related?

**Comment by xeroine** [comment, marked as answer]
Ugh, I'm sorry, soon after this issue I decided to go back to Debian. You can delete this post if you want/can.
